xroot/Overlay.lua

```diff
@@ -127,13 +142,17 @@ local function endFloatingMenu()
     Timer.cancel(floatingMenuTimer)
     floatingMenuTimer = nil
   end
-  floatingMenuObject = nil
```

Comment:
This doesn't _seem_ to be necessary. Having it here though prevents the "fire" methods I added from working though since the reference is gone.